gril/grilunsol.c

```diff
@@ -345,6 +368,8 @@ struct ril_data_call_list *g_ril_unsol_parse_data_call_list(GRil *gril,
 	g_free(raw_addrs);
 	g_free(raw_dns);
 	g_free(raw_gws);
+	if (raw_pcscf != NULL)
+		g_free(raw_pcscf);
```

Comment:
On discussion with you, it appears raw_pcscf is not freed in anything but the error case.

Reply:
That should be fixed.